wiki/math/gaussian-process-gaussian-mixture-model.md

```diff
@@ -0,0 +1,172 @@
+---
+# Jekyll 'Front Matter' goes here. Most are set by default, and should NOT be
+# overwritten except in special circumstances. You should set the article's title:
+title: Guassian Process and Gaussian Mixture Model
+# The 'title' is automatically displayed at the top of the page
+# and used in other parts of the site.
+---
+
+This document acts as a tutorial on Gaussian Process(GP), Gaussian Mixture Model, Expectation Maximization Algorithm. We recently ran into these approaches in our robotics project that having multiple robots to generate environment models with minimum number of samples. 
+
+You will be able to have a basic understanding on Gaussian Process (what is it and why it is very popular), Gaussian Mixture Model and Expectation Maximization Algorithm after this tutorial.
+
+# Table of Contents
+1. [Gausssian Process](#Gausssian-Process)
+3. [Gaussian Mixture Model](#Gaussian-Mixture-Model)
+4. [Expectation Maximization Algorithm](#Expectation-Maximization-Algorithm)
+7. [Summary](#Summary)
+8. [References](#References)
+
+## Gausssian Process
+
+### What is GP?
+A Gaussian Process(GP) is a probability distribution over functions \\( p(\textbf{f}) \\) , where the functions are defined by a set of random function variables \\(\textbf{f} = \{f_1, f_2, . . . , f_N\}\\). For a GP, any finite linear combination of those function variables has a joint (zero mean) Gaussian distribution. It can be used for nonlinear regression, classification, ranking, preference learning, ordinal regression. In robotics, it can be applied to state estimation, motion planning and in our case environment modeling.
+
+GPs are closely related to other models and can be derived using bayesian kernel machines, linear regression with basis functions, infinite multi-layer perceptron neural networks (it becomes a GP if there are infinitely many hidden units and Gaussian priors on the weights), spline models. A simple connections between different methods can be found in the following figure.
+<span style="display:block;text-align:center">![](https://i.imgur.com/HBIqhsR.png)</span>
```

Comment:
You're mixing HTML in with the Markdown here and in the commented out line below. Jekyll doesn't handle these well. These should be stripped. 

Also, images should be put in and assets folder and not rely on third-party sources (such as imgur).